source/Core/Src/settingsGUI.cpp

```diff
@@ -884,29 +884,31 @@ static bool setCalibrateVIN(void) {
   OLED::clearScreen();
 
   for (;;) {
-    OLED::setCursor(0, 0);
+    OLED::setCursor(25, 0);
     uint16_t voltage = getInputVoltageX10(getSettingValue(SettingsOptions::VoltageDiv), 0);
     OLED::printNumber(voltage / 10, 2, FontStyle::LARGE);
     OLED::print(LargeSymbolDot, FontStyle::LARGE);
     OLED::printNumber(voltage % 10, 1, FontStyle::LARGE, false);
     OLED::print(LargeSymbolVolts, FontStyle::LARGE);
+    OLED::setCursor(0, 8);
+    OLED::printNumber(getSettingValue(SettingsOptions::VoltageDiv), 3, FontStyle::SMALL);
 
     switch (getButtonState()) {
     case BUTTON_F_SHORT:
-      nextSettingValue(SettingsOptions::VoltageDiv);
+      prevSettingValue(SettingsOptions::VoltageDiv);
       break;
     case BUTTON_B_SHORT:
-      prevSettingValue(SettingsOptions::VoltageDiv);
+      nextSettingValue(SettingsOptions::VoltageDiv);
       break;
     case BUTTON_BOTH:
     case BUTTON_F_LONG:
     case BUTTON_B_LONG:
       saveSettings();
       OLED::clearScreen();
       OLED::setCursor(0, 0);
-      OLED::printNumber(getSettingValue(SettingsOptions::VoltageDiv), 3, FontStyle::LARGE);
+      warnUser(translatedString(Tr->CJCCalibrationDone), 3 * TICKS_SECOND);
```

Comment:
If we are going to use CJCCalibrationDone for more than one calibration; should probably rename it to just `CalibrationDone`

Reply:
Yes! Of course I'm fine with that.
Since that was just a workaround I did not bother to put effort into that. 😊

Reply:
Done!